src/pyrqlite/cursors.py

```diff
@@ -156,11 +162,14 @@ def execute(self, operation, parameters=None):
         operation = self._substitute_params(operation, parameters)
 
         command = self._get_sql_command(operation)
+        transact = ''
```

Comment:
When automatic transactions are disabled, it probably makes sense to support the `isolation_level` parameter that python's sqlite3 module supports:

https://docs.python.org/3/library/sqlite3.html#sqlite3.Connection.isolation_level

The default behavior, `isolation_level=''`, means to implicitly issue a plain BEGIN statement before a DML statement. The parameter can be set to DEFERRED, IMMEDIATE, or EXCLUSIVE in order to make it implicitly issue the corresponding type of BEGIN statement before a DML statement. For `isolation_level=None`, autocommit mode is triggered, as described here:

https://www.sqlite.org/c3ref/get_autocommit.html